packages/sdk/src/modules/FundOperations.ts

```diff
@@ -39,11 +51,9 @@ export function withFundOperations<TBase extends MidasBaseConstructor>(Base: TBa
       const currentSignerAddress = await this.signer.getAddress();
 
       const hasApprovedEnough = (await token.callStatic.allowance(currentSignerAddress, cTokenAddress)).gte(amount);
-      if (!hasApprovedEnough) {
-        const max = BigNumber.from(2).pow(BigNumber.from(256)).sub(constants.One);
-        const approveTx = await token.approve(cTokenAddress, max);
-        await approveTx.wait();
-      }
+      // if (!hasApprovedEnough) {
```

Comment:
If code is commented out without an explanation it should be just removed. We can restore everything from the git history if ever needed again.